.gitignore

```diff
@@ -168,3 +168,76 @@ Temporary Items
 static/*
 
 */static/css/*.css
+
```

Comment:
Lets use `gitignore.io` for something that covers both front-end and back-end like: https://www.gitignore.io/api/vim,node,venv,emacs,linux,macos,python,django,virtualenv,intellij+all,visualstudiocode

Reply:
@kilemensi front end additions to `.gitignore` was created using `gitigonre.io`. But I had to add `**/**/` because frontend folder is at two level down the root project

Reply:
What didn't work @KhadijaMahanga? My understanding is if `bin` is in `.gitignore`, `bin` file or directory will be ignored at **any** level below the current directory.